azure-devops/rtd-projects/06_pipeline_devops-sender-auth.tf

```diff
@@ -0,0 +1,132 @@
+variable "rtd-ms-sender-auth" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "rtd-ms-sender-auth"
+      branch_name     = "refs/heads/main"
+      pipelines_path  = ".devops"
+      yml_prefix_name = "cstar-rtd"
+    }
+    pipeline = {
+      enable_code_review = true
+      enable_deploy      = true
+      path               = "rtd\\rtd-ms-sender-auth"
+    }
+  }
+}
+
+locals {
+  # global vars
+  rtd-ms-sender-auth-variables = {
+    dockerfile = "Dockerfile"
+  }
+  # global secrets
+  rtd-ms-sender-auth-variables_secret = {
+
+  }
+  # code_review vars
+  rtd-ms-sender-auth-variables_code_review = {
+    sonarcloud_service_conn = "SONARCLOUD-SERVICE-CONN"
+    sonarcloud_org          = var.rtd-ms-sender-auth.repository.organization
+    sonarcloud_project_key  = "${var.rtd-ms-sender-auth.repository.organization}_${var.rtd-ms-sender-auth.repository.name}"
+    sonarcloud_project_name = var.rtd-ms-sender-auth.repository.name
+  }
+  # code_review secrets
+  rtd-ms-sender-auth-variables_secret_code_review = {
+
+  }
+  # deploy vars
+  rtd-ms-sender-auth-variables_deploy = {
+    K8S_IMAGE_REPOSITORY_NAME        = replace(var.rtd-ms-sender-auth.repository.name, "-", "")
+    DEPLOY_NAMESPACE                 = local.domain
+    SETTINGS_XML_RW_SECURE_FILE_NAME = "settings-rw.xml"
+    SETTINGS_XML_RO_SECURE_FILE_NAME = "settings-ro.xml"
+    HELM_RELEASE_NAME                = var.rtd-ms-sender-auth.repository.name
+
+    DEV_CONTAINER_REGISTRY_SERVICE_CONN = local.service_endpoint_azure_devops_docker_dev_name
+    DEV_KUBERNETES_SERVICE_CONN         = local.srv_endpoint_name_aks_dev
+    DEV_CONTAINER_REGISTRY_NAME         = local.aks_dev_docker_registry_name
+    DEV_AGENT_POOL                      = local.azdo_agent_pool_dev
+
+    UAT_CONTAINER_REGISTRY_SERVICE_CONN = local.service_endpoint_azure_devops_docker_uat_name
```

Comment:
@petretiandrea just for safety we may disable integration with uat and prod envs